src/pages/ReimbursementAccount/BusinessInfo/substeps/AddressBusiness.tsx

```diff
@@ -49,7 +39,15 @@ const validate = (values) => {
     return errors;
 };
 
-function AddressBusiness({reimbursementAccount, onNext, isEditing}) {
+type AddressBusinessOnyxProps = {
+    reimbursementAccount: OnyxEntry<ReimbursementAccount>;
+};
+
+type AddressBusinessProps = {
+    reimbursementAccount: ReimbursementAccount;
+} & SubStepProps;
```

Comment:
Also please, move types to the top of the file.
```suggestion
type AddressBusinessOnyxProps = {
    reimbursementAccount: OnyxEntry<ReimbursementAccount>;
};

type AddressBusinessProps = AddressBusinessOnyxProps & SubStepProps;
```